src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetTaskByIdControllerCFTTest.java

```diff
@@ -149,7 +149,7 @@ public void should_return_a_200_and_retrieve_a_task_by_id_jurisdiction_location_
             .body("task.case_management_category", equalTo("Protection"))
             .body("task.work_type_id", equalTo("decision_making_work"))
             .body("task.permissions.values.size()", equalTo(5))
-            .body("task.permissions.values", hasItems("Read", "Refer", "Own", "Manage", "Cancel"))
+            .body("task.permissions.values", hasItems("Read", "Own", "Manage", "Cancel"))
```

Comment:
this same `hasItems("Read", "Own", "Manage", "Cancel")` is called several times here ? It could be encapsualted within a private method ?